resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -216,10 +222,13 @@ private[spark] class ExecutorPodsAllocator(
         execPods(execId) = execPodState
       }
     }
-
-    var totalPendingCount = 0
+    var sumPendingPods = 0
```

Comment:
Can we reuse the old variable `totalPendingCount`? It looks resemble enough for `sumPendingPods` in the next context.